src/iop/grain.c

```diff
@@ -462,6 +556,9 @@ void commit_params(struct dt_iop_module_t *self, dt_iop_params_t *p1, dt_dev_pix
   d->channel = p->channel;
   d->scale = p->scale;
   d->strength = p->strength;
+  d->midtones_bias = p->midtones_bias / 100.0f;
```

Comment:
If for consistency with the old code in this IOP, then yes, probably better to handle it in callbacks.
But i **think** in new code it should be done in `commit_params()` if needed at all